src/add-hooks.coffee

```diff
@@ -12,6 +12,8 @@ sandboxHooksCode = require './sandbox-hooks-code'
 mergeSandboxedHooks = require './merge-sandboxed-hooks'
 HooksWorkerClient = require './hooks-worker-client'
 
+# Ensure platform agnostic path.basename function
+basename = if process.platform is 'win32' then path.win32.basename else path.basename
```

Comment:
This works as intended, the problem was in `cli-test.coffee` - fixed by https://github.com/apiaryio/dredd/pull/803/commits/5273b321aee06c08c7fa8a00dad1f3f815c89f61.